src/rules/colon_assignment_spacing.coffee

```diff
@@ -38,14 +38,12 @@ module.exports = class ColonAssignmentSpacing
         nextToken = tokenApi.peek 1
 
         checkSpacing = (direction) ->
+            offset = -1
             spacing =
                 switch direction
                     when 'left'
-                        token[2].first_column - previousToken[2].last_column - 1
+                        token[2].first_column - previousToken[2].last_column + offset
```

Comment:
Is there a reason we need the `offset` variable if it always `-1`?

Reply:
> csx tags 'column' resolves to the beginning of the tag definition, rather than the '<'

previously coffescript didn't seem to parse it correctly for csx/jsx tags, now its fixed. so just moved it to a variable, nothing much.

Reply:
Could you change it to `-1` and remove the `offset` variable since we never change the offset?

Reply:
done.